test/test.js

```diff
@@ -0,0 +1,99 @@
+/* global describe, it */
+
+// const assert = require('assert')
+const chai = require('chai')
+const utils = require('../index')
+const path = require('path')
+const manifest = require(path.resolve('package.json'))
+
+chai.should()
+chai.expect()
+
+describe('napi-build-utils', function () {
+  it('manifest validates', function () {
```

Comment:
You may consider moving this to a `before()` function to separate it from the rest of the tests. It's not a huge deal, but it makes it clear that it's ensuring a sane test environment and not actually testing the module code itself. Further, if the "before all" hook fails, none of the tests will run, so you won't see any "false negatives" (that is, tests that fail due to the environment being wrong, not something with the module).

Reply:
👍🏼